onnx_xla/operator_registry.cc

```diff
@@ -41,6 +43,36 @@ namespace onnx_xla  {
     return ONNXIFI_STATUS_SUCCESS;
   }
   REGISTER_OPERATOR_TRANSLATOR(Relu, translateRelu)
+
+  //To be tested
+  //If 1 input, match output value to same op input value is matched to
+  //Otherwise do successive additions, being sure to satisfy broadcast semantics
+    //If same rank or scalar, no need to specify
+    //otherwise, use numpy matching of trailing dimensions
+  onnxStatus translateSum(const Node& n, XlaBuilder& builder, ValueOpMap& valueToOp)  {
+    if (n.inputs().size() == 1)  {
+      valueToOp[n.outputs().at(0)] = valueToOp[n.inputs().at(0)];
+      return ONNXIFI_STATUS_SUCCESS;
+    }
```

Comment:
I think you can remove this special case (input_size = 1), your code below is general enough to handle it as well.

Reply:
oh actually, only one input is not even valid in onnx Sum https://github.com/onnx/onnx/blob/master/docs/Operators.md#Sum

Reply:
From what I see, 1 input is allowed: inputs 1 - infinity and in the example

Reply:
oops you are right yeah I need more sleep :-)